### PR TITLE
Lezhnev HW6

### PR DESCRIPTION
**Pull request by Splatjov**
meow

(easy to check via main_test.go though, and all the test files are in 06 directory)

### CODE REVIEW

File: .DS_Store
Comment (kochaika) [inline .DS_Store:1]: This file is not related to your hw

> Before submitting the pull request, make sure that you can say "YES" to each point in this short checklist:
Pull-request changes affect only your folder and only one Home Work.

https://github.com/kochaika/nand2tetris-hw-2024/blob/main/README.md

**Review by kochaika — CHANGES_REQUESTED** [review]

**Comment by Splatjov** [comment]
Done, annoying file is deleted
